app/scienceapi/templates/admin/base_site.html

```diff
@@ -0,0 +1,85 @@
+{% extends "admin/base.html" %}
+{% load mezzanine_tags i18n staticfiles %}
+
+{% block title %}{{ title }} | Mezzanine{% endblock %}
+
+{% block javascripts %}
+<script>
```

Comment:
What's going on ehre?

Reply:
This is mezzanine. I have just overridden a couple of lines below. I wm using `override: start` tags for that